src/Renderer/ThreeExtended/B3dmLoader.js

```diff
@@ -27,9 +31,20 @@ function filterUnsupportedSemantics(obj) {
         }
     }
 }
+// parse for RTC values
+function parseInternalRTC(data, gltf) {
```

Comment:
Each file type has its own RTC mechanism? `CESIUM_RTC` for `b3dm` and `RTC_CENTER` for `pnts`?